<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export DXF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Save To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioRelative">
     <property name="toolTip">
      <string>Moves with the document if moved to a different location. This is better if you share your file with others.</string>
     </property>
     <property name="text">
      <string>Relative to Document</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioAbsolute">
     <property name="toolTip">
      <string>Stays in the same location when the document is moved elsewhere. This is better if you have a directory somewhere of all your outputs.</string>
     </property>
     <property name="text">
      <string>Absolute</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
